client/src/App.js

```diff
@@ -15,13 +15,15 @@ import CreateNft from './components/CreateNft/CreateNft';
 function App() {
   return (
     <div className='App'>
+      <NFTNav></NFTNav>
       <React.Fragment>
-      <Route exact path='/' render={()=><LandingPage/>} />
-      <Route path='/home' render={()=><NFTNav/>} />
-      <Route exact path='/home' render={()=><HomePage/>} />
-      <Route exact path='/createNft' render={()=><CreateNft/>} />
-      <Route exact path='/details/:id' render={({match}) => <Details match={match}/>}/>
-      <Route component={NotFoundException}/>
+        <Switch>
```

Comment:
el switch es importante para que no me renderice el not found exception en todos los path.. sino que lo renderice siempre y cuando no este en ninguno de los path validos declarados